src/main/kotlin/org/malachite/estella/services/SecurityService.kt

```diff
@@ -116,7 +116,9 @@ class SecurityService(
         checkUserRights(jwt, id) && getHrPartnerFromJWT(jwt) != null
 
     fun checkOrganizationHrRights(jwt: String?, hrId: Int) =
-         hrPartnerRepository.findByUserId(hrId).get().organization == getOrganizationFromJWT(jwt)
+        getOrganizationFromJWT(jwt) != null &&
```

Comment:
Ja wiem, że nie idziemy w performance, ale na kiego 3 razy dekodować jwt, skoro można to do zmiennej zapisać????